library/std/src/sys/windows/c.rs

```diff
@@ -1338,6 +1385,7 @@ compat_fn_with_fallback! {
     ) -> NTSTATUS {
         STATUS_NOT_IMPLEMENTED
     }
+    #[cfg(target_arch = "x86")]
     pub fn NtWriteFile(
```

Comment:
Is still still using the dlsym on 32bit Windows? Then we'd have to keep that code in Miri.

Reply:
Oh right! Yeah, we'll need to keep both for now.

Reply:
Hm, should I simply duplicate the code or factor it out somewhere?

Reply:
Please put it in a function somewhere... `windows/foreign_items.rs` seems like a reasonable place for now.